salt/iiif/loris-removal.sls

```diff
@@ -1,3 +1,13 @@
+uwsgi-loris-is-dead:
+    service.dead:
+        - enable: false
+        - name: uwsgi-loris
+
+uwsgi-loris-socket-is-dead:
```

Comment:
didn't know there were two different systemd services. Let's clean up all of them

Reply:
yes, uwsgi by itself has an issue managing it's own socket under systemd that makes restarting the service very very slow. By having systemd manage the socket it all works very nicely.

I found out a bit later that it's possible to manage the systemd socket within the service definition file (I think), so that would save a small bit of configuration overhead. 

With this project I discovered you could have nginx and uWSGI (the project) talk uwsgi (the protocol) over TCP without the need for a socket just by specifying a port. I think there may be a slight overhead to this as I've noticed the APM apdex metrix take a slight dip. There were a lot of other changes happening though.

Reply:
> very very slow. 

iirc it hangs, there is a timeout, and then it's forcibly killed and restarted by systemd.